### ISSUE TITLE
Selectors for Rmd chunk options have ambiguous meaning

### ISSUE THREAD

**Issue by arencambre** [rstudio/rstudio#8462]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.4.1048
    OS Version      : Windows 10
    R Version       : 4.0.3

### Steps to reproduce the problem
In an Rmd, click on the gear icon in a code chunk.

### Describe the problem in detail

Selectors for options are ambiguous. They alternate between left, right, and center. There is no broad and intuitive meaning to how left and right relate to TRUE or FALSE. Center may be slightly less ambiguous, but it's still not a great representation of no setting.

Example dialog:
![image](https://user-images.githubusercontent.com/411495/99996833-ee7afa00-2d81-11eb-87b0-faa0d5e6360f.png)

### Describe the behavior you expected

Clear and intuitive UX for selectors that correspond to values like TRUE and FALSE.

For binary selectors, a checkbox is good often good. However, since you have tri-state selectors (TRUE, FALSE, and no value), that won't work for them. For the sake of consistency, a single type of selector that does both binary and tri-state, and is intuitive, is preferred. For example, a larger selector, where you move a slider to _cover_ the desired state, could be good. That can work for binary (i.e., must select either state) or trinary (where you must select TRUE, FALSE, or no value).

Radio buttons might work, too.

These UX Stack Exchange answers may have viable ideas:

* https://ux.stackexchange.com/a/80802
* https://ux.stackexchange.com/a/80756

- [X] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [X] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [X] If I am reporting a RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@arencambre Thank you for filing the issue! I agree that the UI is a bit hard to follow, including that the center is hard to get just right. I'll file this as an enhancement to consider as we continue development of RStudio:

My suggestion: use radio buttons, with options like "True," "False," and "Default"/"Not Specified"/something similar.

**Comment by arencambre** [comment]
A switch to radio buttons would need careful consideration. A simple switch to radio buttons means you'd be presented with a two-dimensional array of buttons. Without additional hinting, it won't be intuitively clear that the radio buttons require a single selection within each row.

The concept of expanding the sliders to include words that the slider sits above is appealing, if it can be done intuitively. However, I cannot think of an example of where that has been done well.

**Comment by arencambre** [comment]
This is still valid.

**Comment by arencambre** [comment]
Not stale!

**Comment by arencambre** [comment]
Not stale, despite what the mindless, misconfigured robot things.
